packages/components/styles/_responsive.scss

```diff
@@ -0,0 +1,32 @@
+$ca-breakpoint-mobile: 768px;
+$ca-breakpoint-tablet: 1024px;
+
+@mixin ca-media-mobile {
+  @media (max-width: #{$ca-breakpoint-mobile - 1px}) {
```

Comment:
Is this syntax valid? Or should the `}` only be around the variable?

Reply:
Dunno, seems the compiler is happy.